src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -243,12 +247,22 @@ private string CreateNodeAnnotationContent(OpenNodeAnnotationEventArgs e)
             var writer = new StringWriter();
             try
             {
-                // Write the Node info section to the string writer
-                writer.WriteLine(NodeDocumentationHtmlGenerator.FromAnnotationEventArgs(e));
+                writer.WriteLine(DocumentationBrowserUtils.GetContentFromEmbeddedResource(STYLE_RESOURCE));
+
+                // Get the Node info section and remove script tags if any
+                var nodeDocumentation = NodeDocumentationHtmlGenerator.FromAnnotationEventArgs(e);
+                if (MarkdownHandlerInstance.RemoveScriptTagsFromString(ref nodeDocumentation))
```

Comment:
I think this function has a pretty bad name now that we're using sanitizer - can we change it?

Reply:
I agree, I will rename it.